### ISSUE TITLE
New Search dialog issues

### ISSUE THREAD

**Issue by alecpl** [tinymce/tinymce#5082]
There's a new search and replace dialog in 5.0.12. I see some issues there:

1. Keyboard users will not like where Previous/Next buttons are. They should be closer to Replace button. Usually after pressing Find you may want to jump to selected occurrence and replace it. At least I think that's the most common case. Now imagine how many keystrokes you need to navigate between Next and Replace.

2. The .tox-dialog element has box-shadow set, but you can't see it because of its parent overflow style.

3. After update I immediately noticed problem with my custom styles that I added to dialogs. It would be good to distinguish between standard dialog and inline dialog like this one. Maybe the .tox-dialog parent element should have some specific class assigned.

**Comment by TheSpyder** [comment]
The dialog hasn't had full QA yet; community releases are showing off our latest work and I expect this release won't be going out to our enterprise customers until we've iterated on the dialog in 5.0.13.

I just noticed items 4 and 6 myself before I'd finished reading your list, accessibility issues in general will be blockers for an enterprise release.

Item 2 is an issue with our inline dialogs in general that we have scheduled work to track down.

Re item 1, I mostly agree but the TinyMCE 5 dialog design language means that the default action button needs to be blue and I'm not sure if having zero blue buttons would really work. What you're seeing is a bit of a conflict between that requirement and the design this was based on (the dialog in our https://textbox.io editor).

And further regarding 3, I will repeat to you again our supported custom styles are those built using the oxide skin tools. That's all we test with to make sure CSS changes don't break customisations.
